docs/community.md

```diff
@@ -64,103 +27,43 @@ Anyone interested in spatial data infrastructures, libraries, GIS, maps, data cu
 	
 - ### Contribute Code
 		  
-	As open-source software, GeoBlacklight depends on contributions from our developer community. Anyone is welcome to code for the community to review.
+	As open-source software, GeoBlacklight depends on contributions from our developer community. Anyone is welcome to code for review.
 	
 	[:material-github: Read our software contribution guide](https://github.com/geoblacklight/geoblacklight/blob/main/CONTRIBUTING.md)
 		
 - ### Write Documentation
     
-     Help keep our documentation up-to-date or fix information that is missing or unclear.
-
-     [:material-github: Read our website contribution guide](https://github.com/geoblacklight/geoblacklight.github.io/blob/main/CONTRIBUTING.md)
-    
-
-</div>
-
-## Events
+    Help keep our documentation up-to-date or fix information that is missing or unclear.
 
-<iframe src="https://calendar.google.com/calendar/embed?src=geo4lib%40gmail.com&ctz=America%2FChicago" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+    [:material-github: Read our website contribution guide](https://github.com/geoblacklight/geoblacklight.github.io/blob/main/CONTRIBUTING.md)
 
-## Community Roles
-
-### Administrative Roles
-
-**Meeting Facilitators**: Becky Seifried (July 2025–July 2026)
-
-* Manage Google calendar
-* Set up and host monthly meetings 
-* Prepare agendas
-
-**Platform Management Team**: ==2 vacancies==
-
-* Manage GitHub users and roles
-* Manage Google group permissions, roles, etc.
-* Manage Google Drive members
-* Maintain Slack 
-
-### Documentation Roles
-
-**Public Content Manager**: ==1 vacancy==
+ - ### Help Triage Issues
+  
+    Join our Issue Triage Group, where developers meet quarterly to review and prioritize GitHub issues. [Reach out on Slack to join.](https://join.slack.com/t/geo4lib/shared_invite/zt-3baz35i9x-gLVsSGz8MCDlkGePjk7Ejw)
 
-* Keep the public website(s) up to date
-* Write periodic blog posts, including sprint wrap-up posts
+    [:material-github: Review our ongoing project board](https://github.com/orgs/geoblacklight/projects/2) 
 
-**Technical Writing Lead**: ==1 vacancy==
+</div>
 
-* Keep track of what code, metadata, and tutorials need updating
-* Manage GitHub issues related to documentation
-* Coordinate volunteers to update documentation as necessary
 
-### Technical Roles
+## GeoBlacklight Project Maintainers
 
-**Designated Code Mergers**: [see GitHub Teams](https://github.com/orgs/geoblacklight/teams)
+**Designated Code Mergers**: Nick Budak + [see GitHub Teams](https://github.com/orgs/geoblacklight/teams)
```

Comment:
[GitHub Teams](https://github.com/orgs/geoblacklight/teams)

Reply:
I removed the link to the GitHub Teams. Do we still want to refer to them?

Reply:
I think I misspoke during the community call - we don't actually need Nick to be a designated code merger (we *do* still need a Release Coordinator to help Eric though). The Governance Group (us haha) agreed to give the GeoBlacklight Developers team (of which Nick is already a member) the responsibility of code merging. If we make Nick the sole Designated Code Merger now, that will have bigger implications... at the very least, we'll need to also change the [contributing guide](https://github.com/geoblacklight/geoblacklight/blob/main/CONTRIBUTING.md).

I think we should revert this back, but be more specific:

`**Designated Code Mergers**: [GeoBlacklight Developers Team](https://github.com/orgs/geoblacklight/teams/geoblacklight-developers)`

Reply:
I put the link to the GitHub Team back